src/store/movie/operations/getMovieImagesOperation.js

```diff
@@ -0,0 +1,34 @@
+import { createLogic } from 'redux-logic'
+
+import { openNotification } from 'Utils'
+import { storeData } from 'Store/data/actions'
+import { movieSelector } from 'Store/movie/selectors'
+import { GET_MOVIE_IMAGES } from '../types'
+import {
+  requestMovieImagesStart,
+  requestMovieImagesSuccess,
+  requestMovieImagesError
+} from '../actions'
+
+export const getMovieImagesOperation = createLogic({
+  type: GET_MOVIE_IMAGES,
+  latest: true,
+  async process({ action, axios, getState }, dispatch, done) {
+    const { movieId } = action
+    dispatch(requestMovieImagesStart())
+    try {
+      const response = await axios.get(`movie/${movieId}/images`)
+      const images = response.data.backdrops?.slice(0, 3)
```

Comment:
`lodash` `slice` can solve it without additional optional chaining operator

Reply:
@VladWithCookies yes, but I decided to use native methods here